src/styles/components/variables.scss

```diff
@@ -0,0 +1,16 @@
+$day-size: 100px;
+$gap-size: 1px;
+$day-with-gap: $day-size + $gap-size;
+$padding-size: 10px;
+$font-size: 30px;
+$font-family: Arial, sans-serif;
+$max-month-length: 31;
+$days-with-gaps: (
+  mon: 0,
+  tue: $day-with-gap,
+  wed: 2 * $day-with-gap,
+  thu: 3 * $day-with-gap,
+  fri: 4 * $day-with-gap,
+  sat: 5 * $day-with-gap,
+  sun: 6 * $day-with-gap,
```

Comment:
It is better to perform this multiplication logic in the loop itself, so your code will be more readable